src/Fhir.Anonymizer.Core/AnonymizationConfigurations/AnonymizerConfigurationValidator.cs

```diff
@@ -1,140 +1,43 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Text;
 using System.Text.RegularExpressions;
-using Hl7.Fhir.Model;
+using Hl7.FhirPath;
 
 namespace Fhir.Anonymizer.Core.AnonymizerConfigurations
 {
     public class AnonymizerConfigurationValidator
     {
-        private readonly Regex s_fhirPathRegex = new Regex(@"^[a-zA-Z]+(\.([a-zA-Z]+))*$");
-
         public void Validate(AnonymizerConfiguration config)
         {
-            if (config.TypeRules == null && config.PathRules == null)
-            {
-                throw new AnonymizerConfigurationErrorsException("The configuration is invalid, please specify any pathRules or typeRules");
-            }
-
-            var invalidPaths = GetInvalidPaths(config);
-            var invalidTypes = GetInvalidTypes(config);
-            var invalidMethods = GetInvalidMethods(config);
-            var invalidDateShiftTypes = GetInvalidDateShiftTypes(config);
-
-            StringBuilder builder = new StringBuilder(string.Empty);
-            if (invalidPaths.Count > 0)
-            {
-                builder.Append($"The specified path is unsupported: {String.Join(", ", invalidPaths)}. ");
-            }
-            if (invalidTypes.Count > 0)
-            {
-                builder.Append($"The specified type is unsupported: {String.Join(", ", invalidTypes)}. ");
-            }
-            if (invalidMethods.Count > 0)
-            {
-                builder.Append($"The specified method is unsupported: {String.Join(", ", invalidMethods)}. ");
-            }
-            if (invalidDateShiftTypes.Count > 0)
+            if (config.FhirPathRules == null)
```

Comment:
Do we support TypeRules/PathRules after this change? Code from this validator is removed but I see their reference in other files.

Reply:
We would not support TypeRules/PathRules. And update to remove all the reference to TypeRules/PathRules. Thanks